src/routes/blog/post/announcing-init/+page.markdoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Announcing Init, what's next?"
+description: 
+date: 2024-02-19
+cover: /images/blog/init-announcement.png
+timeToRead: 4
+author: laura-du-ry
+category: open-source
+---
+
+We are very excited to finally announce Init. 
+
+A new way to experience Appwrite releases, starting with Appwrite 1.5.
+
+### Celebrating innovation and technology
+
+You bring innovation and technology to life that impacts daily lives. Innovation starts on the brink of something new, and Init is a time to start new ideas, or, even better, initialize new ideas. Whether it be a new project, a contribution, a tutorial, a function, or even a new business, you’re initializing what is next.
+
+During Init, we celebrate and demonstrate new products, features, and functionality for you to build with. We’ve added more content, online events, and other new elements to make Init an event that inspires you to build what is next. 
+
+We also want it to be a fun and welcoming experience for both seasoned community members as well as new joiners. But knowing the open-source community around Appwrite, we don't have to worry about that. 
+
+### Five day long event
+
+Starting on the 12th of February, we will celebrate a new product and/or feature every day and share content to support you in getting started. Alongside, we will host online events each day with creators and friends of Appwrite to learn about the new releases, ask questions, and, most of all, geek out together. To top it all off, we will be hosting a fun closing party that will be held on Discord on Friday, the 16th of February. Find the list of events on Discord.
+
+### A unique ticket
+
+We all know that bringing new ideas to life is hard work. We wanted to showcase this in our special and unique Init tickets that showcase your unique GitHub contribution grid. Those who have contributed to Appwrite have a special edition card as part of our recognition for their support. More variants are possible based on whether you have an Appwrite account and how you customize your ticket. 
+
+Your ticket is also your entry into the raffle to win special edition Init swag.
+
+### Get your ticket and join the fun
+
+Init gives life to new ideas and celebrates them, and we’re excited to see it inspire your next creation. 
+
+Init takes place in the week of 26 February to 01 March 2024. 
+
+"Don't miss out! Be sure to register and claim your ticket at [appwrite.io/init/ticket](appwrite.io/init/ticket)."
+
+
+**"Init_ what’s next?"**
```

Comment:
I like the slogan `Init is the start of something new better` Let's be more consistent with whatever message across all channels.

Reply:
updated